.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,52 @@
+name: Bug report
+description: File a bug report
+title: '[Bug]: '
+labels: [bug]
+assignees: ''
+body:
+  - type: input
+    attributes:
+      label: SDK Version
+      description: |
+        The SDK/npm package version being used when the bug was detected. E.g.: 0.9.3
+      validations:
+        required: false
+  - type: textarea
+    attributes:
+      label: Expected behavior
+      description: |
+        A clear and concise description of what you expected to happen.
+      validations:
+        required: true
+  - type: textarea
+    attributes:
+      label: What is actually happening?
+      description: |
+        A clear and concise description of what the bug is.
+      validations:
+        required: true
+  - type: input
+    attributes:
+      label: Reproduction link
+      description: |
+        A link to real code in jsfiddle.net, codesandbox.io etc is very helpful to speed up bug resolution
+      validations:
+        required: true
+  - type: textarea
+    attributes:
+      label: Steps to reproduce
+      description: |
+      Steps to reproduce the behavior:
+      1. Go to '...'
+      2. Click on '...'
+      3. Scroll down to '...'
+      4. See error
+    validations:
+      required: false
```

Comment:
Shouldn't steps to reproduce be a required field?

Reply:
I was unsure whether it should be required or not, but after thinking some more about it, I concluded it will be better to make this field required than otherwise, so I've changed `required` to `true` as suggested. 

In case someone has a very specific problem which they cannot tell the steps to reproduce (or the steps to reproduce are not useful), they can fill this field with an explanation.